app/src/constants/interfaces/recipe.ts

```diff
@@ -0,0 +1,29 @@
+export interface GitHubResult {
+  name: string;
+  object: {
+    entries: {
+      name: string;
+      object: {
+        text: string;
+      };
+    }[];
+  };
+}
+
+export interface RecipeFile {
+  name: string;
+  content: string;
+}
+
+// TODO: Recipe should have one of readme, config, or html as required
+export interface Recipe {
+  title: string;
+  html: RecipeFile;
+  css: RecipeFile;
+  scss: RecipeFile;
+  readme?: RecipeFile;
+  config: {
```

Comment:
We should create a type for the config? (BTW I think we can call it `cookbook.json`? I prettey like that 😝)